cartoframes/data/observatory/catalog.py

```diff
@@ -1,12 +1,15 @@
 from __future__ import absolute_import
 
+from .dataset import CatalogDataset
 from .entity import is_slug_value
 from .category import Category
 from .country import Country
 from .geography import Geography
-from .dataset import CatalogDataset
+from .subscriptions import Subscriptions
 from .repository.constants import COUNTRY_FILTER, CATEGORY_FILTER, GEOGRAPHY_FILTER
 
+from ...auth import Credentials, defaults
```

Comment:
Why are you importing `defaults` instead of `get_default_credentials`? Just curious

Reply:
This is related with the testing with mocks